### PR TITLE
Privacy Request Capture Interface for Devkit v0.1

### PR DESCRIPTION
**Pull request by jboileau99**
Code and docs for:
- [x] #4 
- [x] #5

### CODE REVIEW

**Comment by jboileau99** [comment]
More work on styling to be done here, but this covers the main requirement of a menu of actions that can be filtered via JSON string passed to the top level component

File: src/DemandActionMenu.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+import './ActionItem.js';
+import './TestElement.js';
+
+/**
+ * Grid menu of demand action types
+ */
+@customElement('demand-action-menu')
+export class DemandActionMenu extends LitElement {
+  @property({ type: String }) title = 'My privacy request';
```
Comment (m4rk055) [inline src/DemandActionMenu.ts:13]: You should integrate [localization](https://lit.dev/docs/localization/overview/) into the design as soon as possible.

File: src/DemandActionMenu.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    /* TODO: Which of these do we want to expose through variables? */
+    .description-heading {
+      color: var(--dmd-actions-menu-decription-color, #000);
+      text-align: left;
+    }
+  `;
+
+  render() {
+    this.exclActions = this.excludeActions.split(',').map(a => a.toLowerCase());
```
[the remark this answers is not in the document]
Comment (jboileau99) [reply src/DemandActionMenu.ts:48]: Not to the top level component as attributes have to be strings, but I should change it to use properties to pass an included actions list down the components

File: src/DemandActionMenu.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    return html`
+      <p class="title-heading">${this.title}</p>
+      <p class="description-heading">${this.description}</p>
+      <div class="actions-container">
+        ${this.actions
+          .filter(a => !this.exclActions.includes(a.NAME.toLowerCase()))
```
[the remark this answers is not in the document]
Comment (jboileau99) [reply src/DemandActionMenu.ts]: Changed to have an includeActions property instead, passed as JSON string

**Comment by jboileau99** [comment]
Closed in favour of #17

**Pull request closed without merging**